tests/bootstrap.php

```diff
@@ -8,29 +8,29 @@
  * For the full copyright and license information, please view the file license.md that was distributed with this source code.
  */
 
-declare(strict_types = 1);
+declare(strict_types=1);
 
 use Tester\Environment;
 
-if (!@include __DIR__ . '/../vendor/autoload.php') {
-	echo 'Install Nette Tester using `composer update --dev`';
-	exit(1);
+if (! @include __DIR__ . '/../vendor/autoload.php') {
+    echo 'Install Nette Tester using `composer update --dev`';
+    exit(1);
 }
 
 // configure environment
 Environment::setup();
 date_default_timezone_set('Europe/Prague');
 
-$_SERVER = array_intersect_key($_SERVER, array_flip([
-	'PHP_SELF',
-	'SCRIPT_NAME',
-	'SERVER_ADDR',
-	'SERVER_SOFTWARE',
-	'HTTP_HOST',
-	'DOCUMENT_ROOT',
-	'OS',
-	'argc',
-	'argv',
+$_SERVER                 = array_intersect_key($_SERVER, array_flip([
+    'PHP_SELF',
+    'SCRIPT_NAME',
+    'SERVER_ADDR',
+    'SERVER_SOFTWARE',
+    'HTTP_HOST',
+    'DOCUMENT_ROOT',
+    'OS',
+    'argc',
+    'argv',
 ]));
 $_SERVER['REQUEST_TIME'] = 1234567890;
-$_ENV = $_GET = $_POST = [];
+$_ENV                    = $_GET = $_POST = [];
```

Comment:
Well this is a bit ugly. :-D

Reply:
😂 Do you mind reporting it to Squizlabs as false positive? It should imho align the last `=`, not first. Sniff `Generic.Formatting.MultipleStatementAlignment`.